sklearn/model_selection/_split.py

```diff
@@ -566,34 +566,34 @@ def _make_test_folds(self, X, y=None, labels=None):
         unique_y, y_inversed = np.unique(y, return_inverse=True)
         y_counts = bincount(y_inversed)
         min_labels = np.min(y_counts)
-        if np.all(self.n_folds > y_counts):
+        if np.all(self.n_splits > y_counts):
             raise ValueError("All the n_labels for individual classes"
                              " are less than %d folds."
```

Comment:
maybe should be `less than n_splits=%d`, although this whole error message could do with better wording.